[App] Add CloudMultiProcessBackend to run an children App within the Flow in the cloud (#15800)
Lightning-AI/lightning: src/lightning_app/CHANGELOG.md

- Added the CLI command `lightning run model` to launch a `LightningLite` accelerated script ([#15506](https://github.com/Lightning-AI/lightning/pull/15506))

> - Added the CloudMultiProcessBackend to run an children App within the Flow in the cloud ([#15800](https://github.com/Lightning-AI/lightning/pull/15800))

Reviewer: ```suggestion
- Added a CloudMultiProcessBackend which enables running a child App from within the Flow in the cloud ([#15800](https://github.com/Lightning-AI/lightning/pull/15800))
```
